app/Main.hs

```diff
@@ -116,6 +116,10 @@ main = do
             let header = "The following entries cannot be copied because a directory already exists at the destination."
             let errorMsgs = NE.toList paths <&> \(from, to) -> "Cannot copy '" +| from |+ "' to '" +| to |+ "'."
             printError $ unlinesF @_ @Builder $ header : "" : errorMsgs
+          CPRParentDirectoryIsEntry paths -> do
+            let header = "The following entries cannot be copied because a destination directory has an entry in it's subpath."
+            let errorMsgs = NE.toList paths <&> \(from, to) -> "Cannot copy '" +| from |+ "' to '" +| to |+ "'."
```

Comment:
I recommend https://hackage.haskell.org/package/nyan-interpolation-0.9/docs/Text-Interpolation-Nyan.html

Reply:
Oh good point! I'll open an issue for this.

Reply:
Done: https://github.com/serokell/coffer/issues/39